styles/venture/layout.s2

```diff
@@ -1217,118 +1228,138 @@ $module_layout_css
 .one-column #content .module-content { text-align: left;}
 
 .two-columns-left #content,
-.three-columns-sides #content { border-left: $*sidebar_width solid transparent; }
+.three-columns-sides #content { border: none; float: none; }
 
-.two-columns-right #content,
-.three-columns-sides #content { border-right: $*sidebar_width solid transparent; }
+.column-right #content { min-width: $*sidebar_width; box-sizing: border-box; }
 
-.three-columns-left #content { border-left: $*sidebar_width_doubled solid transparent; }
+#content { padding: 1em; }
 
-.three-columns-right #content { border-right: $*sidebar_width_doubled solid transparent; }
+@media $medium_media_query {
+    .two-columns-left #content,
+    .three-columns-sides #content { border-left: $*sidebar_width solid transparent; border-right: $*sidebar_width solid transparent; }
 
-.column-right #content { min-width: $*sidebar_width; }
+    .three-columns-left #content { border-left: $*sidebar_width_doubled solid transparent; }
+
+    .two-columns-right #content { border-right: $*sidebar_width solid transparent; }
```

Comment:
I think something similar to this needs to be done but for `.two-columns-left #content` - it's showing up with the space still